src/PlaylistDetails/PlaylistDetails.scss

```diff
@@ -0,0 +1,44 @@
+.playlist-details {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
This will allow for some flexibility if we make the page more responsive down the line.